### PR TITLE
Groupings-941: Fixed all tests except memberAttributesAdminTest()

### PR DESCRIPTION
**Pull request by ruizaj**
Will be creating a ticket to address this final test.

### CODE REVIEW

**Comment by zakgilbert** [comment]
This is a 2.2 ticket so a separate branch needs to be made off of dev-2-2-0.

File: src/test/java/edu/hawaii/its/api/service/TestGroupAttributeService.java [hunk 1]
```diff
@@ -247,17 +247,17 @@ public void changeListServStatusTest() {
         String lastModTime4 = attributes.getWsAttributeAssigns()[0].getWsAttributeAssignValues()[0].getValueSystem();
         assertThat(lastModTime4, is(lastModTime3));
 
-        assertFalse(memberAttributeService.isOwner(GROUPING, username[1]));
+        // Invalid user can't change attribute status
         try {
-            groupAttributeService.changeGroupAttributeStatus(GROUPING, username[1], LISTSERV, true);
+            groupAttributeService.changeGroupAttributeStatus(GROUPING, "zzzzz", LISTSERV, true);
```
Comment (zakgilbert) [inline src/test/java/edu/hawaii/its/api/service/TestGroupAttributeService.java:252]: Is "zzzzz" a possible uhUsername?

**Comment by fduckart** [comment]
Shouldn't there be "fail" calls in those tests in the enableDisablePreferencesFailTest method?

**Comment by mhodgesatuh** [comment]
Question: is 'zzzzzz' potentially a valid uhUuid (UH Number)?  Unlikely, but, technically, yes.  Place and underscore in the middle like this" 'zzz_zzz' and you have picked a username that is not in our namespace given our formatting rules.

File: src/test/java/edu/hawaii/its/api/service/TestGroupAttributeService.java (hunk 1 above)
Comment (mhodgesatuh) [reply src/test/java/edu/hawaii/its/api/service/TestGroupAttributeService.java:252]: Question: is 'zzzzzz' potentially a valid uhUuid (UH Number)? Unlikely, but, technically, yes. Place and underscore in the middle like this" 'zzz_zzz' and you have picked a username that is not in our namespace given our formatting rules.

**Pull request merged**